witchcraft-server/src/minidump/mod.rs

```diff
@@ -61,16 +61,30 @@ pub async fn init() -> Result<(), Error> {
 }
 
 pub fn connect() -> Result<minidumper::Client, Error> {
```

Comment:
This'll need to be an async fn since we're spawning it off into a task and the sleep below use tokio's async sleep.

Reply:
oops